i18n/zh-cn/docusaurus-plugin-content-pages/release-announce/1.4.0.md

```diff
@@ -33,77 +33,65 @@ Apache YuniKorn社区在此版本中修复了270个[JIRAs](https://issues.apache
+现在还可以以文字形式指定日志级别，而不仅仅是数字。文件是 [服务配置](https://yunikorn.apache.org/docs/user_guide/service_config) 的一部分。
 
-Please see [YUNIKORN-1823](https://issues.apache.org/jira/browse/YUNIKORN-1823) for details.
+详细资讯请参阅 [YUNIKORN-1823](https://issues.apache.org/jira/browse/YUNIKORN-1823)
 
-### Removal of the Spark Operator Application Manager
-The Spark operator application manager was added to support the Spark operator as an external integration.
-It added a full lifecycle for the Spark applications submitted via the operator.
-The application manager was never turned on by default and needed a custom configuration to enable it.
+### 移除 Spark Operator 应用程式管理器
+新增了 Spark operator 应用程式管理器以支援 Spark operator 作为外部整合。它为透过 operator 提交的 Spark 应用程式添加了完整的生命周期。
+应用程式管理器从未默认启用，需要自订配置才能启用。
 
-The implementation caused duplicate Spark applications to be created and was partially rolled back in [YUNIKORN-643](https://issues.apache.org/jira/browse/YUNIKORN-643) as part of YuniKorn 0.11.
-The functionality that was left was also covered by the basic pod handling.
-To reduce code maintenance the remaining code is now also removed without the loss of functionality.
+此实作导致创建重复的 Spark application，并在作为 YuniKorn 0.11 的 [YUNIKORN-643](https://issues.apache.org/jira/browse/YUNIKORN-643) 
+中回滚。基本的 Pod 处理涵盖了剩下的功能。为了减少程式码维护，现在也删除了剩余的程式码，而不会失去功能。
 
-Please see [YUNIKORN-2092](https://issues.apache.org/jira/browse/YUNIKORN-2092) for details.
+详细资讯请参阅 [YUNIKORN-2092](https://issues.apache.org/jira/browse/YUNIKORN-2092)
 
-### Removal of the YuniKorn application CRD
-YuniKorn 1.4.0 has removed the application CRD.
-The application CRD was added to allow a simpler integration of applications with YuniKorn.
-This integration never got used and all current integrations rely on simple labels or annotations on the pods.
+### 移除 YuniKorn 应用程式 CRD
+YuniKorn 1.4.0 已移除应用程式 CRD。添加应用程式 CRD 是为了更简单地将应用程式与 YuniKorn 整合。这种整合从未被使用过，目前所有整合都依赖 
+Pod 上的简单标签或是注释。 
 
-Please see [YUNIKORN-1672](https://issues.apache.org/jira/browse/YUNIKORN-1672) for details.
+详细资讯请参阅 [YUNIKORN-1672](https://issues.apache.org/jira/browse/YUNIKORN-1672)
 
-###  Application traceability
-Tracking and troubleshooting applications is specific for YuniKorn.
-Kubernetes does not have an application concept.
-An event system has been added to expose the changes to applications, queues and nodes for external consumption.  
+###  应用程式可追溯性
+追踪和故障排除应用程式是专门针对 YuniKorn。 Kubernetes 中没有应用程式的概念。新增了事件系统来公开应用程式、队列和节点的变更以公外部使用。
 
-Please see [YUNIKORN-1628](https://issues.apache.org/jira/browse/YUNIKORN-1628)
+详细资讯请参阅 [YUNIKORN-1628](https://issues.apache.org/jira/browse/YUNIKORN-1628)
 
-### Build updates
-The YuniKorn builds no longer relies on centrally installed tools while building the binaries or images.
-Each build will download and maintain its own set of tools, not even shared between checked out repositories.
+### 建置更新
+在建置二进位档案或映像时，YuniKorn 建置不再依赖集中安装的工具。每个版本都会下载并维护自己的工具集，甚至不会在签出的除从库之间共用。
```

Comment:
```suggestion
在建置二进位档案或映像时，YuniKorn 建置不再依赖集中安装的工具。每个版本都会下载并维护自己的工具集，甚至不会在签出的储存库之间共用。
```

Reply:
Thank you for your correction, i have corrected it.